torch/_inductor/fx_passes/reinplace.py

```diff
@@ -566,22 +593,20 @@ def tensor_with_same_storage_already_reinplaced(arg):
                     replace_dict[copy_node] = copy_node.args[0]
                 node.target = inplaceable_op.inplace_op
         elif node.target == torch.ops.higher_order.auto_functionalized:
+            from torch._higher_order_ops.auto_functionalize import get_mutable_args
+
             _mutable_op = node.args[0]
-            from torch._higher_order_ops.auto_functionalize import get_mutable_arg_names
+            arg_names, arg_types = get_mutable_args(_mutable_op)
```

Comment:
This line isn't needed anymore?

Reply:
mhmm why linter did not complian about unused variiable

Reply:
There are hundreds of unused local variables in the codebase, unfortunately: I'm working on cleaning most of them up as a spare time project.

The lintrunner toolchain installs `pylint` but never actually calls it, and `pylint` would be the only tool in the toolchain that would actually find these...